zookeeper-api/src/main/java/org/apache/helix/zookeeper/impl/client/FederatedZkClient.java

```diff
@@ -479,8 +479,18 @@ public long getCreationTime(String path) {
 
```

Comment:
Please add API description here and explain where are supported in federatedZK client and in witch cases we throw exception.